packages/core/src/Stores/contract-store.js

```diff
@@ -288,6 +274,7 @@ export default class ContractStore extends BaseStore {
                     this.accu_low_barrier || low_barrier,
                     null,
                     {
+                        color: BARRIER_COLORS.BLUE,
```

Comment:
@ahmadtaimoor-deriv isn't the blue barrier label already implemented by @akmal-deriv separately?

Reply:
It was implemented in SmartCharts package (Master Branch) and implementation with chartiq. 
For Smartcharts , the flutter implementation is in the flutter-chart branch .